src/ucp/core/ucp_worker.c

```diff
@@ -2939,14 +2943,17 @@ ucp_worker_do_keepalive_progress(ucp_worker_h worker)
         ucp_worker_keepalive_next_ep(worker);
     }
 
-    max_ka_ep_count = ucs_min(worker->context->config.ext.keepalive_num_eps,
-                              worker->num_all_eps) - worker->keepalive.ep_count;
+    max_ka_ep_count   = ucs_min(worker->context->config.ext.keepalive_num_eps,
+                                worker->num_all_eps);
+    allow_ka_ep_count = (worker->keepalive.ep_count < max_ka_ep_count) ?
+                                (max_ka_ep_count - worker->keepalive.ep_count) :
+                                0;
```

Comment:
minor: maybe this clang formatting is worth to be reconsidering? Imo, looks weird

Reply:
agree, clang format here doesn't help to make code readable here

Reply:
I'd leave it as is until clang format won't be reconsidered - we have multiple places where the same format has already applied

Reply:
i'd avoid creating more such places
afair, we have a GH issue for tracking such cases
@gleon99, @yosefe wdyt on such format?

Reply:
Yes, we have an issue for the ternary operator alignment:
https://github.com/openucx/ucx/issues/6126#issuecomment-768835389

Reply:
what the suitable format for the ternary operator?
```
allow_ka_ep_count = (worker->keepalive.ep_count < max_ka_ep_count) ?
                    (max_ka_ep_count - worker->keepalive.ep_count) : 0;
```
is it correct?

Reply:
looks good to me

Reply:
IMO better use helper var for readability

Reply:
or ucs_max

Reply:
or the while loop below will be:
`while (worker->keepalive.ep_count < max_ep_count)`